mlir/lib/Conversion/VectorToSPIRV/CMakeLists.txt

```diff
@@ -14,5 +14,6 @@ add_mlir_conversion_library(MLIRVectorToSPIRV
   MLIRSPIRVDialect
   MLIRSPIRVConversion
   MLIRVectorDialect
+  MLIRVectorTransforms
```

Comment:
Should we also add it to the bazel file?

Reply:
Good catch! Should be fixed by the new commit here.